zerver/models.py

```diff
@@ -172,6 +172,7 @@ class Realm(models.Model):
     waiting_period_threshold = models.PositiveIntegerField(default=0)  # type: int
     DEFAULT_MAX_INVITES = 100
     max_invites = models.IntegerField(default=DEFAULT_MAX_INVITES)  # type: int
+    uploads_quota = models.BigIntegerField(null=True)  # type: int
```

Comment:
Let's make this uploads_quota_gb, since it is something humans will need to manipulate.